### PR TITLE
fix(owner): Fix the currently broken `yarn owner` command

### PR DESCRIPTION
**Pull request by rubycut**
**Summary**

* fixing currently broken owner command
* doesn't ask for npm username and password for list command
* adding basic test #3893 

**Test plan**

Yarn now correctly displays the list of package owners. Also added automated tests.

### CODE REVIEW

File: src/cli/commands/owner.js [hunk 1]
```diff
@@ -87,14 +87,11 @@ async function list(config: Config, reporter: Reporter, flags: Object, args: Arr
...
   const name = await getName(args, config);
-
-  reporter.step(1, 3, reporter.lang('loggingIn'));
-  const revoke = await getToken(config, reporter, name);
-
-  reporter.step(2, 3, reporter.lang('ownerGetting', name));
-  const pkg = await config.registries.npm.request(name);
+  reporter.step(1, 1, reporter.lang('ownerGetting', name));
+  const pkg = await config.registries.npm.request(name, {
+    headers: {Accept: 'application/json'},
```
Comment (arcanis) [inline src/cli/commands/owner.js]: Do we have to specify the headers? Isn't `config.registries.npm` doing it for us?

Comment (rubycut) [reply src/cli/commands/owner.js]: I tried without headers and it didn't work, and such json headers appear in other commands also.

Comment (BYK) [reply src/cli/commands/owner.js]: @arcanis - maybe we can add this by default in a separate diff if it is not done already?

**Comment by BYK** [comment]
@rubycut can you merge from master and see if that fixes the broken builds?

**Comment by rubycut** [comment]
@BYK , I rebased to master, nothing changed. 

macos checks always fail (free plan?), appveyor also fails, which doesn't seem to be connect with patch, appveyor error message:

Build started
git clone -q https://github.com/yarnpkg/yarn.git C:\projects\yarn
git fetch -q origin +refs/pull/5593/merge:
git checkout -qf FETCH_HEAD
An error occurred while sending the request.

**Comment by BYK** [comment]
>macos checks always fail (free plan?), appveyor also fails, which doesn't seem to be connect with my pull request, appveyor error message:

This doesn't happen on master or on other pull requests. Something is up with how this is set up. I'll see what's wrong. For some reason both services are trying to use your account instead of Yarn's account.

**Comment by BYK** [comment]
Thanks a lot!

**Comment by rubycut** [comment]
@BYK when I started development, I opened my own circle-ci account, because I was working on enterprise tests, maybe that was the issue. 

If I fork your repo, can I automatically run tests on circle-ci during development in forked repo without opening account there? I doubt that. 

So, what is is the proper workflow?

**Comment by BYK** [comment]
@rubycut - ah, that probably explains it. You can't use Yarn's CircleCI account unless you submit a PR to it. Once a PR is opened, CircleCI should run that specific PR in Yarn's account. I'll let them know of this issue.

Thanks for the help!

**Pull request merged**